### ISSUE TITLE
Failed to activate the atom-minify package

### ISSUE THREAD

**Issue by kliker** [armin-pfaeffle/atom-minify#15]
When I open Atom.app, I am getting "Cannot set property 'shouldDisplay' of null" message.

**Atom Version**: 1.0.19
**System**: Mac OS X 10.10.5
**Thrown From**: [atom-minify](https://github.com/armin-pfaeffle/atom-minify) package, v0.4.2
### Stack Trace

Failed to activate the atom-minify package

```
At Cannot set property 'shouldDisplay' of null

TypeError: Cannot set property 'shouldDisplay' of null
    at Object.module.exports.registerContextMenuItem (/Users/aleks/.atom/packages/atom-minify/lib/atom-minify.coffee:322:37)
    at Object.module.exports.activate (/Users/aleks/.atom/packages/atom-minify/lib/atom-minify.coffee:253:19)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:245:19)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:226:30
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:169:15)
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:218:14)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:486:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:469:29
    at Config.module.exports.Config.transact (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:324:16)
    at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:464:19)
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-light-ui",
      "base16-tomorrow-dark-theme"
    ],
    "disabledPackages": [
      "project-palette-finder"
    ]
  },
  "atom-minify": {}
}
```
### Installed Packages

``` coffee
# User
atom-cli-diff, v1.1.0
atom-minify, v0.4.2
file-type-icons, v0.7.3
git-tab-status, v1.9.2
jsonlint, v1.1.2
language-htaccess, v0.1.1
project-manager, v2.6.5
remote-atom, v1.2.7
symbols-tree-view, v0.10.0
tabs-to-spaces, v0.11.1
tree-view-git-projects, v0.5.0

# Dev
No dev packages
```

**Comment by armin-pfaeffle** [comment]
Hey! That is strange... In my menu file (`atom-minify.cson`) there is an entry that adds a menu item in the context menu of the tree view. A function shows or hides the item on special files, configurable via option [Show Minify-item in Tree View context menu](https://github.com/armin-pfaeffle/atom-minify#show-minify-item-in-tree-view-context-menu).

So, I don't see any reason, why the item should be null... Please try the following: close Atom, open terminal, enter the following command and retry Atom.

```
apm rebuild-module-cache
```

Does this help you?

**Comment by kliker** [comment]
Rebuilding cache didn't help. I also tried to: 
1. uninstall/install package
2. change Tree View context menu to both "On every file" and "No"

but I am still getting the same message on Atom start.

**Comment by armin-pfaeffle** [comment]
Can you upload the following files of the atom-minify directory?! There must be an error in them...
- `lib/atom-minify.coffee`
- `menus/atom-minify.cson`

You can find them in `/Users/<username>/.atom/packages/atom-minify`.

**Comment by kliker** [comment]
I compared `/Users/<username>/.atom/packages/atom-minify` with [atom-minify-0.4.2](https://github.com/armin-pfaeffle/atom-minify/archive/v0.4.2.zip) and they are identical (other than package.json and node_modules).

**Comment by armin-pfaeffle** [comment]
Oh... Perhaps I know why this error occurs. I mixed something up with different branches and published versions. I will fix that today or tomorrow!

**Update**: Yep, I'm right! I could reconstruct that bug! I will fix it with next version.

**Comment by armin-pfaeffle** [comment]
Hey! Sorry that my update is so late. I published version 0.5.0 right now, which fixes some bugs and you problem too! Please tell me, if it's working for you!?

**Comment by kliker** [comment]
You fixed it, thanks!
